### PR TITLE
all: smoother notification dark mode (fixes #4424)

### LINKED ISSUE #4424: dashboard: notification back button is not working
### Description

The back button on notifications is not working.
User has to manually navigate to the dashboard

### Demo

https://github.com/user-attachments/assets/1836a3e7-cd19-4a97-84ea-4302201dd9ea

### PR DESCRIPTION
**Pull request by deeppp15**
## Description

(fixes #4424)
- Removed two files which were from previous notification changes
- We had duplicate row notifications and notification fragment.

## Code Changes
- Removed the old files 
- Used exisiting spineer item instead of default android.

## Screenshot

image

### CODE REVIEW

**Review by Okuro3499 — CHANGES_REQUESTED** [review]
all filter is not visible
Screenshot_20240919_145843

**Comment by deeppp15** [comment]
> all filter is not visible Screenshot_20240919_145843

Hi @Okuro3499 ,

Could you try invalidating cache and reviewing again. I dont see it on my end.

https://github.com/user-attachments/assets/ea3cabb6-236d-4fd3-bd1d-6155358f0077

File: app/src/main/res/layout/fragment_notification.xml [hunk 1]
```diff
@@ -14,25 +14,22 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent">
-        
+
```
Comment (dogi) [inline app/src/main/res/layout/fragment_notification.xml:17]: oops

**Comment by Okuro3499** [comment]
> > all filter is not visible Screenshot_20240919_145843
> 
> Hi @Okuro3499 ,
> 
> Could you try invalidating cache and reviewing again. I dont see it on my end. @Kuljeet1998 are you also seeing the same UI as Gideon ?
> 
>  Notification.changes.mp4

@deeppp15 I still have the issue
[nit.webm](https://github.com/user-attachments/assets/e7bb3e90-88d5-4213-84e8-530788e0f820)

**Comment by Kuljeet1998** [comment]
@deeppp15 Yes, I don't see that either.
image

**Comment by deeppp15** [comment]
Hi @Okuro3499  @Kuljeet1998 

I have tried my changes on multiple devices they are working fine here is an image from another emulator 
image
.

- Make sure you are pulling changes after checking out.

**Review by Okuro3499 — CHANGES_REQUESTED** [review]
@deeppp15 I can now see the filter. make the dropdown button visible in darkmode

**Review by Okuro3499 — APPROVED** [review]
Screenshot_20240924_171502

**Review by Kuljeet1998 — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**